### ISSUE TITLE
Streamparse 3.0.1 - "sparse run" error if more than one environment in config.json

### ISSUE THREAD

**Issue by kateabbasi** [Parsely/streamparse#304]
In previous version of streamparse, I did not have to specify _--environment_ when executing **sparse run**.  However, in 3.0.1, when I do specify the environment, I get the following error:

> topo$ sparse run --environment 'local'
error: Found more than one environment in config.json.  When more than one environment exists, you must explicitly specify the environment name via the -e or --environment flags.

My config.json has two environments: "dev" and "local"

**Comment by dan-blanchard** [comment]
This is intentional. Since we now support specifying options to your topology in many different places (`config.json` file, `Topology.config` attribute, `Component.spec()`'s `config` parameter), we need to know which environment you're using to find all your settings. Without this, running `sparse run` could have your topology have blank settings that it needed to function properly, if you specified them in `config.json`.

**Comment by kateabbasi** [comment]
Thanks for the quick reply! I understand the need for config.json with sparse run. However, if more than one environment is specified in config.json, I get this error:

**error: Found more than one environment in config.json. When more than one environment exists, you must explicitly specify the environment name via the -e or --environment flags.**

I did explicitly specify the environment:

**sparse run --environment 'local'**
